Foundation/Unit.swift

```diff
@@ -234,13 +294,33 @@ open class UnitAcceleration : Dimension {
     
     public required init?(coder aDecoder: NSCoder) { super.init(coder: aDecoder) }
     open override func encode(with aCoder: NSCoder) { super.encode(with: aCoder) }
+
+    open override func isEqual(_ object: Any?) -> Bool {
+        guard let other = object as? UnitAcceleration else {
+            return false
+        }
+
+        if self === other {
+            return true
+        }
+
+        return super.isEqual(object)
+    }
 }
 
 open class UnitAngle : Dimension {
     
     /*
      Base unit - degrees
      */
+
+    public required init(symbol: String) {
```

Comment:
If both of these methods call into super, neither needs to be overridden and can just be inherited (defining the convenience initializer below won't affect anything).

Reply:
Of course! I have no idea why I assumed otherwise 😶

Reply:
Hehe, that's alright, happens to the best of us 🙂